packages/terra-paginator/src/Paginator.module.scss

```diff
@@ -6,61 +6,83 @@
     display: flex;
     justify-content: center;
 
-    &.pageless {
-      justify-content: space-between;
-    }
-
-    &.progressive {
-      justify-content: space-between;
-
-      .nav-link.previous {
-        margin-left: var(--terra-paginator-progressive-nav-link-previous-margin-left, 0.14285rem);
-        margin-right: var(--terra-paginator-progressive-nav-link-previous-margin-right, 0.14285rem);
-      }
-
-      .nav-link.next {
-        margin-left: var(--terra-paginator-progressive-nav-link-next-margin-left, 0.14285rem);
-        margin-right: var(--terra-paginator-progressive-nav-link-next-margin-right, 0.14285rem);
-      }
+    // html input[type='button'].nav-link selector used to
+    // prevent a WebKit bug destroys native `audio` and `video` controls
+    // in Android 4. More info:
+    // https://github.com/necolas/normalize.css/issues/20
+    // https://code.google.com/p/android/issues/detail?id=34301
+    // appearance: button; set to correct the inability to style
+    // clickable types in iOS and Safari.
+    // Appearance needs to be applied only to button/input elements, as
+    // applying to anchor element will cause render issue.
+    // More info:
+    // https://github.com/necolas/normalize.css/issues/566
+    // https://github.com/necolas/normalize.css/commit/1be44f3b2a79af5d463c7ea0264fee0e6fa464ca
+    // stylelint-disable selector-no-qualifying-type
+    button.nav-link,
+    input[type='button'].nav-link,
+    input[type='reset'].nav-link,
+    input[type='submit'].nav-link {
```

Comment:
We can remove `input[type='reset'].nav-link` and `input[type='submit'].nav-link` selectors

Reply:
Nice catch, grabbed the full reset from terra-button without cleaning it up for this usecase.

Reply:
Resolved with https://github.com/cerner/terra-core/pull/2485/commits/a8bac0f3a5011a5bfe3cff8a50012ef539a822b0